packages/extract-react-types/converters-typescript.test.js

```diff
@@ -601,6 +601,23 @@ const TESTS = [
 
     export default MyComponent;
   `
+  },
+  {
+    name: 'memo wrapping forwardRef, with inferred types',
+    typeSystem: 'typescript',
+    code: `
+    import React, { forwardRef, memo } from 'react';
+
+    type MyComponentProps = {
+      foo: string,
+    }
+
+    const MyComponent = memo(forwardRef<HTMLElement, MyComponentProps>((props, ref) => {
```

Comment:
Will `forwardRef` also works without generics? Like this:

```ts
const MyComponent = memo(forwardRef((props: MyComponentProps, ref: React.Ref<HTMLElement>) => {
```

Reply:
Hey! Yeah that should work, there's a test case for this above I'm pretty sure 😀

Reply:
Might be in the flow tests actually

Reply:
Nice. @madou: You have any comments? Looks good to me overall.